src/vs/workbench/contrib/comments/browser/commentsViewActions.ts

```diff
@@ -168,3 +189,32 @@ registerAction2(class extends ViewAction<ICommentsView> {
 		view.filters.showResolved = !view.filters.showResolved;
 	}
 });
+
+registerAction2(class extends ViewAction<ICommentsView> {
+	constructor() {
+		super({
+			id: `workbench.actions.${COMMENTS_VIEW_ID}.toggleSortByUpdatedAt`,
+			title: localize('toggle sorting by updated at', "Sort by Updated At"),
+			category: localize('comments', "Comments"),
+			icon: Codicon.history,
+			viewId: COMMENTS_VIEW_ID,
+			toggled: {
+				condition: ContextKeyExpr.equals('commentsView.sortBy', CommentsSortOrder.UpdatedAtDescending),
+				title: localize('sorting by updated at', "Sort by Updated At"),
+			},
+			menu: {
+				id: MenuId.ViewTitle,
+				when: ContextKeyExpr.equals('view', COMMENTS_VIEW_ID),
+				group: 'navigation',
+				order: 0,
+				isHiddenByDefault: false,
+			},
```

Comment:
I actually like what you had before (where it was in the filter menu) better!

Reply:
And I finally found a precedent for sorting:

![image](https://github.com/user-attachments/assets/b62b8dce-c0bd-4c3b-a683-7578b122acb8)

Reply:
You should be able to add a submenu with something like this:
```
export const commentSortSubmenu = new MenuId('submenu.filter.commentSort');
MenuRegistry.appendMenuItem(viewFilterSubmenu, {
	submenu: commentSortSubmenu,
	title: localize('comment sorts', "Sort By"),
	group: 'navigation',
	icon: Codicon.filter,
	when: ContextKeyExpr.equals('view', COMMENTS_VIEW_ID),
});
```